indico/web/client/js/react/util/Overridable.jsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {connect} from 'react-redux';
+
+
+function Overridable({children, overrides, id, ...restProps}) {
+    const childProps = children.props;
+    const child = React.Children.only(children).type;
+    const Override = overrides[id || child];
```

Comment:
is overriding by actual object useful? if a plugin import such an object it probably won't be `===` anymore, since webpack will include the actual object in the plugin's codebase.

=> `child.getComponentName()`?

Reply:
Yeah, that's a bug indeed.